pages/candidates.tsx

```diff
@@ -1,14 +1,18 @@
 ﻿import React from "react";
 import {NextPage} from "next";
-import textStyle from "../pageStyles/text-classes.module.scss"
+import textStyle from "../pageStyles/text-classes.module.scss";
+import scss from "../pageStyles/candidates.module.scss";
 import Layout from "../components/Layout/layout";
 import {CandidateList} from "../components/CandidateList/CandidateList";
+import Link from "next/link";
 
 const Candidates: NextPage = () => {
 
     return (
         <Layout>
             <h1 className={textStyle.pageHeader}>Candidate Manager</h1>
+            <button className={scss.desktopButton}><Link href={"/create"}><p className={scss.link}>+ Create Candidate</p></Link></button>
```

Comment:
I recon we might be able to do this without having 2 buttons...
How about
```tsx
<Link href="/create">
    <a>+<span> Create Candidate</span></a>
</Link>
```

Then we can just style the `Create Candidate` label to only appear on desktop?

Reply:
Just one now!